src/CheckoutSdk/Payments/Request/Source/Apm/RequestSepaSource.cs

```diff
@@ -0,0 +1,25 @@
+using Checkout.Common;
+
+namespace Checkout.Payments.Request.Source.Apm
+{
+    public class RequestSepaSource : AbstractRequestSource
+    {
+        public RequestSepaSource() : base(PaymentSourceType.Sepa)
+        {
+        }
+
+        public CountryCode Country { get; set; }
```

Comment:
Please add ? to enums, dates, primitives, so that is not default during runtime.
e.g `CountryCode?`

Reply:
👍